src/Form/LegacyConsumer/Commands/Helper/AttributeBag.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Give\Form\LegacyConsumer\Commands\Helper;
+
+class AttributeBag {
+	public function __construct( $attributes = [] ) {
+		foreach ( $attributes as $key => $value ) {
+			$this->$key = $value;
+		}
+	}
+}
```

Comment:
I'm not a huge fan of undeclared, dynamic properties on classes as we can run into the limitations of properties. The simple solution is to have a `private $bag` property which the `$attributes` gets assigned to, then use `__isset()` and `__get()` magic methods.

Another option, which my actually be faster, is instead of having a class for this, just do the following where you'd use this class:
```php
new AttributeBag(['foo' => 'bar']);

// becomes
(object)['foo' => 'bar']; // creates an StdClass with the key-value pairs set
```